tests/libmonitoring/prometheus.go

```diff
@@ -212,19 +213,16 @@ func DoPrometheusHTTPRequest(cli kubecli.KubevirtClient, endpoint string) []byte
 }
 
 func getPrometheusURLForOpenShift() string {
-	var host string
+	virtClient := kubevirt.Client()
 
-	Eventually(func() error {
-		var stderr string
+	var route *ocpv1.Route
+	Eventually(func(g Gomega) {
 		var err error
-		host, stderr, err = clientcmd.RunCommand(testsuite.GetTestNamespace(nil), clientcmd.GetK8sCmdClient(), "-n", "openshift-monitoring", "get", "route", "prometheus-k8s", "--template", "{{.spec.host}}")
-		if err != nil {
-			return fmt.Errorf("error while getting route. err:'%v', stderr:'%v'", err, stderr)
-		}
-		return nil
-	}, 10*time.Second, time.Second).Should(BeTrue())
+		route, err = virtClient.RouteClient().Routes("openshift-monitoring").Get(context.Background(), "prometheus-k8s", metav1.GetOptions{})
+		g.Expect(err).ToNot(HaveOccurred())
+	}, 10*time.Second, time.Second).Should(Succeed())
```

Comment:
This can simply be:
```golang
	Eventually(func() err {
		var err error
		route, err = kubevirt.Client().RouteClient().Routes("openshift-monitoring").Get(context.Background(), "prometheus-k8s", metav1.GetOptions{})
		return err
	}, 10*time.Second, time.Second).Should(Succeed())
```